docs/options.rst

```diff
@@ -189,6 +189,36 @@ The date format, combination of d, dd, D, DD, m, mm, M, MM, yy, yyyy.
 * M, MM: Abbreviated and full month names, respectively.  Eg, Jan, January
 * yy, yyyy: 2- and 4-digit years, respectively.  Eg, 12, 2012.
 
+Object.
+
+Custom formatting options
```

Comment:
Add an empty line between this text and the bullets. Then the bullets will be rendered correctly!